src/components/Skeleton/styles.module.scss

```diff
@@ -0,0 +1,21 @@
+@import '../../scss/variables';
+
+.skeleton {
+  animation: pulse 1s infinite ease-in-out;
+  border-radius: $border-radius;
+  height: 100%;
```

Comment:
The skeleton should always be the same height as the text it's replacing so I don't think `100%` is a good value for text.

How about aiming to simulate the select box more closely, like this?

```diff
diff --git a/src/components/Skeleton/styles.module.scss b/src/components/Skeleton/styles.module.scss
index 197d0a7..c87798a 100644
--- a/src/components/Skeleton/styles.module.scss
+++ b/src/components/Skeleton/styles.module.scss
@@ -3,7 +3,6 @@
 .skeleton {
   animation: pulse 1s infinite ease-in-out;
   border-radius: $border-radius;
-  height: 100%;
   min-height: 1rem;
   width: 100%;
 }
diff --git a/src/components/VersionSelect/index.tsx b/src/components/VersionSelect/index.tsx
index 953540e..bd3edcf 100644
--- a/src/components/VersionSelect/index.tsx
+++ b/src/components/VersionSelect/index.tsx
@@ -1,4 +1,5 @@
 import * as React from 'react';
+import makeClassName from 'classnames';
 import { Col, Form } from 'react-bootstrap';
 // eslint-disable-next-line import/no-unresolved
 import { FormControlProps } from 'react-bootstrap/lib/FormControl';
@@ -69,7 +70,12 @@ class VersionSelectBase extends React.Component<PublicProps> {
           <Form.Label>{label}</Form.Label>
 
           {isLoading ? (
-            <div className="form-control">
+            <div
+              className={makeClassName(
+                'form-control',
+                styles.simulatedFormControl,
+              )}
+            >
               <Skeleton />
             </div>
           ) : (
diff --git a/src/components/VersionSelect/styles.module.scss b/src/components/VersionSelect/styles.module.scss
index eae3a48..0ce2103 100644
--- a/src/components/VersionSelect/styles.module.scss
+++ b/src/components/VersionSelect/styles.module.scss
@@ -2,3 +2,8 @@
   margin: auto;
   padding-top: 14px;
 }
+
+.simulatedFormControl {
+  align-items: center;
+  display: flex;
+}
```

The skeleton still needs a width adjustment (see the other comment) but I think this layout transition is smoother because the text appears in the same place as the skeleton:

<img width="792" alt="Screenshot 2019-04-17 11 53 13" src="https://user-images.githubusercontent.com/55398/56306204-8b39d800-6107-11e9-8648-658a8804bbac.png">
<img width="791" alt="Screenshot 2019-04-17 11 53 28" src="https://user-images.githubusercontent.com/55398/56306205-8b39d800-6107-11e9-991d-70fb17710eec.png">

Reply:
I was thinking about embedding the skeleton component inside the container of the thing it is supposed to mimic, hence 100% but I did not try all the possibilities.

What you're saying is still relevant of course.

Reply:
So, if I remove `height: 100%`, then the `Skeleton` does not fit the container's dimensions, which I think is a problem.

Reply:
Ok, I found a way to combine both idea:

<img width="1392" alt="Screen Shot 2019-04-18 at 10 41 07" src="https://user-images.githubusercontent.com/217628/56348343-8425cf00-61c6-11e9-8059-eb8c22f3a5ba.png">